pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```

Comment:
Why is the version downgraded

Reply:
Maybe it something wrong in my local pnpm, I've rollback it,